### PR TITLE
acpi: flesh out non-normative portion

### PR DESCRIPTION
**Pull request by andreiw**
This should cover the non-server specific portions of https://github.com/riscv-non-isa/riscv-acpi/blob/master/riscv-acpi-guidance.adoc

Still need to cross-check the Illustrative OS-A-SEE Specification if there was anything else interesting there.

Also, APEI/RAS is muddy. Need to understand if there is APEI without RAS.

### CODE REVIEW

File: acpi.adoc [hunk 1]
```diff
@@ -16,8 +16,8 @@ A compliant system must:
 ** Multiple APIC Description Table (MADT, cite:[ACPI] Section 5.2.12)
 ** RISC-V Hart Capabilities Table (RHCT, cite:[ACPI] Section 5.2.36)
 * Meet the requirements in <<Minimum Hardware Requirements Imposed By ACPI>>
-* Be 64-bits clean.
-** RSDT must not be implemented.
+* [[acpi-64bit-clean]]Be 64-bits clean.
+** RSDT must not be implemented, with RsdtAddress in RSDP be 0.
```
Comment (jmcneill-vmware) [inline acpi.adoc:20]: `RSDP be 0` -> `RSDP set to 0`

File: non-normative/acpi.adoc [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+[[acpi-guidance-pcie]]
+===== PCIe
+
+On some architectures, it became an industry accepted norm to describe PCIe implementations not compliant to the PCI Firmware Specification cite:[PCIFW]
+using specification-defined ACPI tables and objects. RISC-V systems compliant to the BRS must only expose ECAM-compatible implementations using the
+MCFG and the standard AML Hardware ID (HID) PNP0A08 and Compatible ID (CID) PNP0A03, and must not rely on ACPI table header information or other out-of-band
+means of detecting quirked behavior.
+
+Some minor incompatibilities, such as split PCIe configuration spaces or incorrect CFG0 filtering, can be handled by emulating ECAM accesses in privileged firmware
```
Comment (jmcneill-vmware) [inline non-normative/acpi.adoc:82]: Starting with pcifw 3.3, split PCIe configuration spaces are supported by MCFG (multiple entries for a single segment are permitted so long as the bus number ranges do not overlap).

File: acpi.adoc [hunk 3]
```diff
@@ -16,8 +16,8 @@ A compliant system must:
 ** Multiple APIC Description Table (MADT, cite:[ACPI] Section 5.2.12)
 ** RISC-V Hart Capabilities Table (RHCT, cite:[ACPI] Section 5.2.36)
 * Meet the requirements in <<Minimum Hardware Requirements Imposed By ACPI>>
-* Be 64-bits clean.
-** RSDT must not be implemented.
+* [[acpi-64bit-clean]]Be 64-bits clean.
```
[the remark this answers is not in the document]
Comment (andreiw) [reply acpi.adoc:19]: Ok. Done.

File: non-normative/acpi.adoc (hunk 2 above)
Comment (andreiw) [reply non-normative/acpi.adoc]: thanks

File: acpi.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (adurbin-rivos) [reply acpi.adoc:19]: Nit: this is referencing a non-normative part of the doc. We'll probably want to put explicit in the normative section explaining the requirement.

**Comment by adurbin-rivos** [comment]
We can land this and iterate on things. Feel free to decide which path forward is best.

**Pull request merged**